sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryHttpPolicy.java

```diff
@@ -95,21 +100,21 @@ public HttpPipelinePolicy create() {
 
     private static void addSpanRequestAttributes(Span span, HttpRequest request,
         HttpPipelineCallContext context) {
-        putAttributeIfNotEmptyOrNull(span, SemanticAttributes.HTTP_USER_AGENT,
+        putAttributeIfNotEmptyOrNull(span, HTTP_USER_AGENT,
```

Comment:
Was `SemanticAttributes` removed?

Reply:
>removes semconv dep, as guidance by oTel that the versioning/module changes for semconv might not be in convergence with oTel and can be managed independently (not stable)

OTel moved Semantic Attributes to its separate module where it will independently be evolved and so could have potential breaking changes.